src/app/interfaces/ProductItemDetailed.ts

```diff
@@ -0,0 +1,8 @@
+export interface ProductItemDetailed {
```

Comment:
**suggestion:** follow the angular convention for naming files https://angular.io/guide/styleguide#general-naming-guidelines. This should something like: `product-item-detailed.ts`